src/components/Input/Input.scss

```diff
@@ -1,28 +1,47 @@
 @import '~om';
 
 .Input {
-  @include text(body1);
-  line-height: 1;
-  padding: spacing(3, 4);
-  border: 1px solid color(elephant);
-  background-color: color(snow);
+  .Input__input {
```

Comment:
`&__input` shorthand will compile this into a flat (non-nested) rule. This is currently compiled as `.Input .Input__input` which is fine but less optimal here.

Reply:
Oh I see it's because you want the modifiers with the ampersand syntax? One way of doing this is using interpolation. We can leave this alone for now and deal with later.

Reply:
(Feel free to merge this whenever you're ready, btw)

Reply:
Good catch actually, this is another left over artifact from changing around the component structure. I will just do &__input here.